app/controllers/idv/in_person/address_controller.rb

```diff
@@ -43,6 +46,24 @@ def extra_view_variables
         }
       end
 
+      # update Idv::DocumentCaptureController.step_info.next_steps to include
+      # :ipp_address instead of :ipp_ssn in delete PR
```

Comment:
I think we talked about this but want to check that my understanding is correct. next_step should be the first non-FSM step. So currently- that is SSN. Upon delete- it will be address.

But still can't remember why not just keep it ipp_ssn b/c it is the next step in the flow

Reply:
The in person flow has DocumentCapture -> (several React steps) -> State ID -> Address (optional) -> SSN -> Verify Info. Eventually FlowPolicy will have :document_capture next_steps include :ipp_state_id when that's out of the FSM. We might need to wait to update it until then, actually, since Address is optional. It's going to take some fiddling to get all that right.